Presentation.Web/app/components/it-system/it-system-catalog.controller.ts

```diff
@@ -711,17 +779,139 @@
             return regexp.test(Url.toLowerCase());
         };
 
-        // show usageDetailsGrid - takes a itSystemUsageId for data and systemName for modal title
-        public showUsageDetails = (usageId, systemName) => {
-            //Filter by usageId
-            this.usageGrid.dataSource.filter({ field: "ItSystemId", operator: "eq", value: usageId });
+        // show usageDetailsGrid - takes a itSystemId for data and systemName for modal title
+        public showUsageDetails = (systemId, systemName) => {
+            this.resetMigrationFlow(); //Migration flow starts freshly from this function
+            this.oldItSystemName = systemName;
+            //Filter by systemId
+            this.usageGrid.dataSource.filter({ field: "ItSystemId", operator: "eq", value: systemId });
             //Set modal title
-            this.modal.setOptions({ title: `Anvendelse af ${systemName}` });
+            this.modal.setOptions({
+                close: (_) => true,
+                resizable: false,
+                title: `Anvendelse af ${systemName}`
+            });
             //Open modal
             this.modal.center().open();
         }
 
-        // usagedetails grid
+        private resetMigrationFlow = () => {
+            this.newItSystemObject = null;
+            this.municipality = null;
+            this.oldItSystemUsageId = null;
+            this.oldItSystemName = null;
+            this.resetItSystemSelection();
+        }
+
+        private resetItSystemSelection = () => {
+            this.convertToSelect2Object("#new-system-usage").select2("data", null);
```

Comment:
this.convertToSelect2Object("#new-system-usage") går igen i begge funktioner og er i sig selv en funktion der ikke er lavet endnu --> `getItSystemSelectionControl()`